### PR TITLE
Finalizes install.sh tails script

### LINKED ISSUE #1316: Finalizing install.sh setup script for Tails environment
During a final run-through of the new Tails upgrade docs (see #1314), specifically to test how the new `tails_files/install.sh` script interacts with upgraded Tails devices (#1309), encountered a few issues that should be cleaned up prior to release.

Fortunately the outstanding issues are cosmetic, and do not require functional changes to the script. In sum, the terminal output displayed after running the script was confusing. Let's refactor to make the output more readable.

**Note:** All Onion URLs have been redacted from messages below, even though were generated exclusively for testing. No other output from the script has been modified before including below.

## Silence zenity noise
When zenity needs to prompt for user input (for Admins, the SSH username on the servers; for Journalists, the Document Interface ATHS info and Source URL, if not defined), the terminal displays some ugly warnings about GUI toolkits:

```
amnesia@amnesia:~/Persistent/securedrop/tails_files$ sudo ./install.sh 
[sudo] password for amnesia: 
Found Document Interface ATHS info in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-document-aths'...
Creating SSH aliases for SecureDrop servers in ~/.ssh/config...
GtkDialog mapped without a transient parent. This is discouraged.

** (zenity:28513): WARNING **: atk-bridge: Couldn't listen on dbus server: Failed to bind socket "/root/.cache/at-spi2-socket-28513": No such file or directory
Found Document Interface ATHS info in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-document-aths'...
Successfully configured Tor and set up desktop bookmarks for SecureDrop!
You will see a notification appear on your screen when Tor is ready.
```

The `Gtk-Message: ` and `WARNING` lines should not be displayed. This can be handled by redirecting STDERR to /dev/null in all calls to zenity. **Only STDERR should be redirected;** STDOUT is captured by the function return and therefore should output normally.

## Verbose logging messages obscure output
During review of #1309, we decided to add verbose messages to let Admins know where the relevant A/THS URLs were found, since the script checks a variety of locations. Unfortunately the post-review implementation doesn't handle interpolation well, because it's unintelligently appending to STDOUT:

```
The Document Interface's Tor onion URL is: http://<REDACTED>.onion
The Source Interfaces's Tor onion URL is: http://Found Source Interface Onion URL in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-source-ths'...
<REDACTED>.onion

The App Server's SSH hidden service address is: <REDACTED>.onion
The Monitor Server's SSH hidden service address is: <REDACTED>.onion
SSH aliases are set up. You can use them with 'ssh app' and 'ssh mon'.
```

See how the `Found Source Interface` line is improperly smack dab in the middle of the line? That's no good. The problem can be partially resolved by printing the verbose logging messages to STDERR, but they're still displayed multiple times unnecessarily:

```
amnesia@amnesia:~/Persistent/securedrop/tails_files$ sudo ./install.sh 
[sudo] password for amnesia: 
Found Document Interface ATHS info in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-document-aths'...
Found Source Interface Onion URL in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-source-ths'...
SSH config already exists, not overwriting...
Found Document Interface ATHS info in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-document-aths'...
Found Source Interface Onion URL in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/app-source-ths'...
Successfully configured Tor and set up desktop bookmarks for SecureDrop!
You will see a notification appear on your screen when Tor is ready.

The Document Interface's Tor onion URL is: http://<REDACTED>.onion
The Source Interfaces's Tor onion URL is: http://<REDACTED>.onion

The App Server's SSH hidden service address is: <REDACTED>.onion
The Monitor Server's SSH hidden service address is: <REDACTED>.onion
SSH aliases are set up. You can use them with 'ssh app' and 'ssh mon'.
```

Rather than hack away at a better solution, I'm just going to remove those verbose messages for now. 

## Succinct checklist
Because progress bars are great:

* [x] Silence zenity noise
* [x] Verbose logging messages obscure output

### PR DESCRIPTION
**Pull request by conorsch**
Implements eleventh hour cleanup for the new `tails_files/install.sh` script (originally provided in #1309). Briefly:

* removes verbose logging messages to avoid cluttering terminal output
* silences STDERR for `zenity` prompts to avoid cluttering terminal outut
* preserves informative messaging around needing to prompt, but redirects to STDERR rather than STDOUT.

See #1316 for verbose explanations of the rationale behind these changes. Closes #1316.

### CODE REVIEW

File: tails_files/install.sh [hunk 1]
```diff
@@ -209,23 +209,20 @@ function lookup_document_aths_url()
...
     app_document_aths="$(grep ^HidServAuth /etc/tor/torrc | head -n 1 | awk '{ print $2 }')"
   # Last shot before prompting: check for an existing desktop icon specifically
   # for the Document Interface. If found, we can extract the URL from there.
   elif [ -e "${amnesia_desktop}/document.desktop" ] ; then
-    echo "Found Document Interface ATHS info in '${amnesia_desktop}/document.desktop'..."
     app_document_aths="$(grep ^Exec=/usr/local/bin/tor-browser | awk '{ print $2 }')"
   # Couldn't find it anywhere. We'll have to prompt!
   else
-    echo "Could not find Document Interface ATHS info, prompting interactively..."
+    echo "Could not find Document Interface ATHS info, prompting interactively..." 1>&2
```
Comment (garrettr) [inline tails_files/install.sh]: What does this change do, and why?

Comment (ageis) [inline tails_files/install.sh]: Redirects output to STDERR...

Comment (conorsch) [inline tails_files/install.sh]: By redirecting the verbose logging message to STDERR, we ensure that the echoed text does not interfere with any of the variable lookups. All of the lookup functions use STDOUT capture (via `$()`) to return config information. This is mostly a defensive measure to ensure that outputs are not mixed.

**Comment by conorsch** [comment]
Fixed the zenity commit and rebased on top of `release/0.3.7`. My commits in the issue and PR about the necessity of redirecting only STDERR and not STDOUT were on point. During review, @garrettr noticed that not all calls to zenity were following that pragma.

File: tails_files/install.sh (hunk 1 above)
Comment (garrettr) [inline tails_files/install.sh]: @conorsch Thanks for the explanation, very helpful.

**Pull request merged**